validation/src/test/scala/hmda/validation/rules/lar/macro/SimplifiedMacroSpec.scala

```diff
@@ -0,0 +1,37 @@
+package hmda.validation.rules.lar.`macro`
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+
+trait SimplifiedMacroSpec extends MacroSpec {
+
+  def multiplier: Double
+
+  def relevantLar(lar: LoanApplicationRegister): LoanApplicationRegister
+  def irrelevantLar(lar: LoanApplicationRegister): LoanApplicationRegister
+
+  protected def simplifiedPropertyTests(name: String, multiplier: Double, relevantLar: LoanApplicationRegister => LoanApplicationRegister, irrelevantLar: LoanApplicationRegister => LoanApplicationRegister): Unit = {
```

Comment:
Thanks! And good suggestion. I was having trouble thinking of a good name for it!